src/containers/PrimaryValidatorApi/PrimaryValidatorApiBanks/index.tsx

```diff
@@ -9,19 +9,35 @@ const PrimaryValidatorApiBanks: FC = () => {
 
       <DocEndpoint endpoint="/banks" method="GET" />
       <RequestResponseSnippet
-        code={`[
-  {
-    "account_number": "5e12967707909e62b2bb2036c209085a784fabbc3deccefee70052b6181c8ed8",
-    "ip_address": "192.168.1.232",
-    "node_identifier": "d5356888dc9303e44ce52b1e06c3165a7759b9df1e6a6dfbd33ee1c3df1ab4d1",
-    "port": 8000,
-    "protocol": "http",
-    "version": "v1.0",
-    "default_transaction_fee": "1.0000000000000000",
-    "confirmation_expiration": null,
-    "trust": "0.00"
-  }
-]`}
+        code={`{
+    "count": 20,
```

Comment:
Does the count value need to be changed to "count":2, as we are showing 2 results in snippet?

Reply:
Hi! no, the count is the total number of records. Ideally, we would want to show the user the next object as well so if you hit the URL like this
http://143.110.137.54/banks?limit=2
it will give 2 records and we will have the next object as well so the user can know how to fetch more records if required since we are just showing 2. 

Also, some PRs have Param Table added for example [here](https://github.com/thenewboston-developers/Website/pull/981/files) but it's not yet final whether we are going to include the Query Params in the docs or not.

Reply:
Hi! no, the count is the total number of records. Ideally, we would want to show the user the next object as well so if you hit the URL like this
http://143.110.137.54/banks?limit=2
it will give 2 records and we will have the next object as well so the user can know how to fetch more records if required since we are just showing 2. 

Also, some PRs have Param Table added for example [here](https://github.com/thenewboston-developers/Website/pull/981/files) but it's not yet final whether we are going to include the Query Params in the docs or not.